### PR TITLE
Release/2.9

### PR DESCRIPTION
**Pull request by v-anshie**
- Added templates for bug reports, feature requests, and pull requests to adhere to adhere with Github's [recommended community standards](https://opensource.guide).
- Added support for owner/group in the plist resource. Allows for plist files to be created under a specific owner. Defaults to root/wheel for compatibility with earlier versions of the cookbook. ([Issue #51](https://github.com/Microsoft/macos-cookbook/issues/51))
- Added support for setting the mode property when creating a plist using the `plist` resource. This allows control over setting the file permissions. ([Issue #51](https://github.com/Microsoft/macos-cookbook/issues/51))

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 All notable changes to this project will be documented in this file.
+## [2.9.0] - 2018-12-06
+### Added
+- Added templates for bug reports, feature requests, and pull requests to adhere to adhere with Github's [recommended community standards](https://opensource.guide).
```
Comment (mjmerin) [inline CHANGELOG.md]:
```suggestion
- Added templates for bug reports, feature requests, and pull requests to adhere with Github's [recommended community standards](https://opensource.guide).
```

**Comment by chrismarlin** [comment]
It might be helpful to add unit or integration testing to validate the behavior of modifications to the user or group of an existing plist.

**Comment by jeroenj** [comment]
Could it be that there was something wrong releasing this to the Chef supermarket? Or does merging this not necessarely mean that the new version is being released?

**Comment by jazaval** [comment]
Sorry about that @jeroenj. 2.9.0 is now available on the Chef Supermarket. https://supermarket.chef.io/cookbooks/macos/versions/2.9.0

**Comment by jeroenj** [comment]
Thanks! 🙇‍♂️

**Pull request merged**